pycontw/pycon.tw
# feat(sponsor): add diamond level

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Add "Diamond" into sponsor levels in the backstage.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to path/admin/sponsors/sponsor/
2. Check if level "Diamond"/"鑽石級" is in the list

## Expected behavior
<img width="150" alt="image" src="https://github.com/user-attachments/assets/5f8dadff-310f-4fde-803d-85b1e3174e0f">

## More Information
Discord message: https://discord.com/channels/752904426057892052/1231150103360835585/1266414865489330238
